.rubocop.yml

```diff
@@ -48,9 +48,15 @@ Metrics/AbcSize:
 Metrics/BlockLength:
   Enabled: false
 
+Metrics/CyclomaticComplexity:
+  Enabled: false
```

Comment:
That's a pretty good rule imo and I'd have kept it.
If it's to avoid errors coming from migrations, imo we shouldn't even lint migration files at all as they're mostly auto-generated.
```yaml
AllCops:
  NewCops: enable
  Exclude:
    - "bin/**/*"
    - "config/**/*"
    - "db/**/*"
    - "node_modules/**/*"
    - "tmp/**/*"
    - "vendor/**/*"
```

Reply:
Agreed. I strongly hesitated with that option, I will create a new PR 👍

Reply:
#536 :)